### ISSUE TITLE
Remove unverified bad email addresses. 

### ISSUE THREAD

**Issue by UXPrime** [chrisflesher/rcvfortexas#96]
Remove unverified bad emails of contacts in the database. Contacts with bad emails but additional  modes of communication can remain.

**Comment by hwasserstrum** [comment]
I think a lot of the bad emails are from sign-up sheets that were entered in the back office. The project should first list the supporters with bad emails and indicate the tag that shows where they were recruited. We can send that list to the person who assigns entry people and see if they can double check. That might get us a good email. If not, if we have a phone number we should text the person, tell them we couldn't read what they entered for their email, and ask them to send an address which we need when we want them to contact their legislators. See what Austin Begin did on an earlier project.
Thanks

**Comment by UXPrime** [comment]
I actually discovered after exploring the 'People' category, and figuring out a way to confirm duplicates, I realized that a number of the flagged emails were already corrected and entered as a separate entry thus becoming duplicates that seemed incomplete. So I merged those and the emails were updated as a result. At this point. We have only one duplicate, which I cannot merge because the name belongs to an admin. @miketbrand
